backend/app/db/alembic_dev/versions/74e82392f17f_make_difficulty_nullable.py

```
@@ -0,0 +1,34 @@
+"""make difficulty nullable
+
+Revision ID: 74e82392f17f
+Revises: a4d2ee11c335
+Create Date: 2025-01-16 09:00:49.714662
+
+"""
+from typing import Sequence, Union
+
+from alembic import op
+import sqlalchemy as sa
```

Ruff (0.8.2)

`sqlalchemy` imported but unused

Remove unused import: `sqlalchemy`

(F401)